### ISSUE TITLE
Enabling AF Location Groupings

### ISSUE THREAD

**Issue by drfaustusfade** [pfitzpaddy/ngm-reportDesk#366]
Based on 2020 reporting, estimation is that around 10 percent :monocle_face: of reports would benefit from that feature

**Comment by pfitzpaddy** [comment]
Originally the concept was to enable the user to dynamically add a list of groups and then assign a group to a location. In the monthly report, that group would then be rendered as a card. Currently it’s automatically grouped by admin2 if I’m not mistaken.

**Comment by pfitzpaddy** [comment]
One idea is to add a location group as a card and then add locations within that card when adding projects within the project details

**Comment by drfaustusfade** [comment]
that will be nice functionality; having group card will look nice, but if a change of location group is needed they will need to delete/add location, save we have drag and drop between groups, 
also, grouping by admin1,2 will be easy for common users, and custom grouping for advanced ones

**Comment by pfitzpaddy** [comment]
Agreed. Grouping by admin1 or admin2 is a good / easier start. Step 2 might look at adding a location group card, which enables the location group title to be updated, and, the location can be assigned to different locations groups via a simple drop down. At all times at least one default location group card would be required in this custom approache. Step 3 could be direct drag and drop. The easiest way is always the best, maybe step 2 and 3 is overkill.... at least for now
